src/api/settings.js

```diff
@@ -1,13 +1,11 @@
-import { getInstance } from 'd2'
-import { onError } from './index'
-
 const SYSTEM_SETTINGS = ['keyGatherAnalyticalObjectStatisticsInDashboardViews']
 
-export const apiFetchSystemSettings = () => {
-    const endPoint = '/systemSettings'
-    const url = `${endPoint}?${SYSTEM_SETTINGS.map(s => `key=${s}`).join('&')}`
-
-    return getInstance()
-        .then(d2 => d2.Api.getApi().get(url))
-        .catch(onError)
+const query = {
+    systemSettings: {
```

Comment:
Wherever I used a query definition I ended up not putting the namespace in it.
In that way, whoever uses the query does not have to know how the namespace is called, but can decide what to use.
For example see:
https://github.com/dhis2/analytics/blob/ac78f355fc5712e1bddf0e90a390155731a9752a/src/api/dimensions.js#L8-L15
https://github.com/dhis2/analytics/blob/ac78f355fc5712e1bddf0e90a390155731a9752a/src/api/dimensions.js#L233-L239

Reply:
Updated